### DISCUSSION TITLE
Pinned and Recommended Showing Back up

### DISCUSSION THREAD

**Discussion by cobimaestro** [valinet/ExplorerPatcher#3193] [General]
So EP has been such a nice aesthetic since I installed W11...because I hate the new start menu design.  Ever since getting EP and swapping to a W10 layout, the Reccommended and Pinned section has been non-existent, until today.  Literally this morning everything was fine, but when I got back home later in the day it looks like the image again.  

I tried uninstalling, reinstalling and making sure it was set to Windows 10, but for some reason it just won't change back.  

I hate having to click All Apps everytime I wanna open the start menu, and sure I could simply put my most used ones on the pinned, or type in quickly what I want to open, but I've been so used to doing it the usual way for years now I'd rather just live in normalcy 

![Untitled](https://github.com/valinet/ExplorerPatcher/assets/168318161/881cb171-e1f5-4197-9a02-6117e19daa31)

**Comment by askinner432** [comment]
This happened to me as well after 2024-04 Cumulative Update Preview for Windows 11 Version 23H2 for x64-based Systems (KB5036980)

**Reply by AKomie** [reply]
The Cumulative Update Preview broke EP for me as well. I backed out the update reinstalled EP and it works. Let the update install and broke it again.

**Comment by pyrates999** [comment]
Install the latest preview release of EP to fix it not using the windows 10 start menu.

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.

**Comment by TheRiftreaper** [comment]
Any ETA for when this will be fixed?
